language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -133,47 +143,18 @@ tests damlTypes yarn damlc daml2ts davl = testGroup "daml2ts tests"
         buildProject []
       withCurrentDirectory here $ do
         step "daml2ts..."
-        setupWorkspace
-        (exitCode, _, err) <- readProcessWithExitCode daml2ts ([groverDar, superGroverDar] ++ ["-o", daml2tsDir, "-p", here </> "package.json"]) ""
+        copyDirectory damlTypes "daml-types"
+        writePackageJson
```

Comment:
Why not make
```haskell
setupWorkspace = do
  copyDirectory
  writePackageJson
```
Maybe under the name `setupYarnEnvironment` or something similar?

Reply:
Ok